packages/components/src/button/index.tsx

```diff
@@ -131,7 +143,10 @@ export function UnforwardedButton(
 		'is-small': size === 'small',
 		'is-compact': size === 'compact',
 		'is-tertiary': variant === 'tertiary',
-		'is-pressed': isPressed,
+
+		'is-pressed': ariaPressed && ariaPressed !== 'false',
```

Comment:
Could we simplify this check?

```suggestion
		'is-pressed': ariaPressed !== 'false',
```

Reply:
The problem here is that `ariaPressed` has a boolean-ish value; it can either be `true`/`"true"`, `false`/`"false"`, `"mixed"` or `undefined`. So we can't _just_ test for `"false"`, because that would give false positives for `false` and `undefined`.

Reply:
If we wanted to be a bit more explicit, an alternative might be...
```js
[true, "true", "mixed"].contains(ariaPressed)
```

Reply:
That variety of value possibilities is a good case for adding some unit tests, by the way.

Reply:
Thank you for the explanation, @andrewhayward ! For which specific implementation to use, I'll leave it up to you to decide which one you think is the clearest to understand for other developers coming across this component.

> That variety of value possibilities is a good case for adding some unit tests, by the way.

I guess we could add a bunch of extra tests checking for `false`, `'true'` and `'false'` values too

Reply:
Added a few extra tests, and altered the logic around this to be a bit more explicit about what is expected.